### DISCUSSION TITLE
Vercel Analytics started recording some bot responses recently

### DISCUSSION THREAD

**Discussion by CurtisGreen** [vercel/community#3587] [Bug Report]
### Page to Investigate

https://vercel.com/my-user/my-project/analytics

### Steps to Reproduce

TLDR: Vercel analytics used to properly filter our bot page views but starting somewhere between August 2nd-3rd they are no longer all filtered out.

I have an integration setup with Checkly which will ping various pages of my site to check for performance. Previously this would not be shown in Vercel Analytics but as of early August they are tracked and displayed as visitors/page views.

The easiest way to demonstrate this is to show the last 30 days without any filters and then with the OS GNU/Linux filter:

No filters:
![image](https://github.com/vercel/community/assets/9402065/9f6c995e-95e9-404e-9fdb-72de9bf73a60)

OS is GNU/Linux:
![image](https://github.com/vercel/community/assets/9402065/e130b89b-930c-4eec-bf28-5060210afa5a)

There are some real users under GNU/Linux but the majority of them are the constant stream of bot page views. After checking the requests I can confirm that the pages visited were the exact ones I have it set to check

**Comment by filmic** [comment]
User agent string used by Checkly:
`Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.5845.82 Safari/537.36 (Checkly, https://www.checklyhq.com)`
